packages/eslint-plugin/src/rules/no-useless-template-literals.ts

```diff
@@ -110,7 +115,12 @@ export default createRule<[], MessageId>({
         }
 
         const fixableExpressions = node.expressions.filter(
-          (expression): expression is TSESTree.Literal | TSESTree.Identifier =>
+          (
+            expression,
+          ): expression is
```

Comment:
consider just returning boolean/omitting the return type since the type guard isn't needed for any subsequent code.